ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/e2e/96066_TL_legal-requirements/105392_legal-requirements.cy.js

```diff
@@ -1,13 +1,14 @@
 /// <reference types ='Cypress'/>
 
 // TO DO: Check Legal Requirement validation on first time use; check Empty tags.
+import legalRequirements from "../../pages/legalRequirement"
 
-const url = Cypress.env('url') + '/task-list/2054'
-
-describe('Legal Requirements', { tags: ['@dev', '@stage']}, () => {
+describe('Legal Requirements', { tags: '@dev'}, () => {
     beforeEach(() => {
-        cy.visit(url)
-        cy.get('[data-cy="select-tasklist-links-legalrequirements"]').click()
+        legalRequirements.selectProject().then(() =>{
+            cy.url()
```

Comment:
What is the cy.url() used for?

Reply:
Its a function that returns the method of locating conversion, project(0).

Reply:
Good catch! Updating it now.